src/i18n/locale/en.json

```diff
@@ -339,7 +340,13 @@
         "reader_settings": "Reader Settings"
       }
     },
-    "title": "Reader - Manga {{mangaId}} Chapter {{chapterIndex}}"
+    "title": "Reader - Manga {{mangaId}} Chapter {{chapterIndex}}",
+    "error": {
+      "label": {
+        "unable_to_find_next_chapter": "Unable to find prev chapter - \"Skip duplicate chapters option\" is enabled",
```

Comment:
Are these translations correct? next/prev. Also, use full words (previous) in messages.

Reply:
oh, yeah, the translations are mixed up